event_types/event_types.py

```diff
@@ -696,7 +733,7 @@ def bin_data_in_energy_and_offset(
             10 ** log_e_reco_bins[i_e_bin - 1], 10 ** log_e_high
         )
         if len(this_dtf_e) < 1:
-            raise RuntimeError("The range {} is empty".format(this_e_range))
+            warnings.warn("The range {} is empty".format(this_e_range))
```

Comment:
I assume it makes sense for you to now have it just print a warning instead of raising an error in this case?

Reply:
Yes, we don't want it to stop if a bin is empty, but it's still useful to know if there are any empty bins, so a warning should be good enough.